Checkout/Checkout/Source/Model/Card+Scheme.swift

```diff
@@ -138,8 +138,9 @@ extension Card {
 
     var maxCardLength: Int {
       switch self {
+      case .visa:
+          return 19
```

Comment:
i am wondering about this logic, it should allow 19 or 16 or 19 only?

Reply:
Most Visa cards are 16 digits. Apparently they can also be 19 characters long. So both 16 and 19 are valid, and the regex seems to already support 16 and 19. The only issue was with allowing input to reach 19.

Reply:
Yep - looks okay, as that variable sets the _maxCardLength_, which in VISA's case, is indeed 19.